clar2wasm/tests/wasm-generation/tuple.rs

```diff
@@ -0,0 +1,47 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+use crate::PropValue;
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_tuple(
+        key in "[a-zA-Z]{1}([0-9][a-zA-Z]|[a-zA-Z][0-9]){1,20}".prop_map(|k| k),
+        expr_1 in (1..10usize).prop_flat_map(PropValue::any_sequence))
+    {
+        crosscheck_compare_only(
+            &format!("(tuple ({key} {expr_1}))")
+        )
+    }
+}
```

Comment:
This is less efficient than generating a tuple directly and evaluating it, which is what `values::evaluated_value_is_the_value_itself` already does.

Reply:
Ok. Let's ignore this test since the `tuple` functionality is already covered.